libnest/src/config.rs

```diff
@@ -72,17 +92,17 @@ impl Config {
     ///
     /// ```
     /// # extern crate libnest;
+    ///
```

Comment:
Blank line (the `#extern crate libnest` will not be shown in the documentation because it's starts with a `#`)